taxi/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from taxi.views import index
+
+urlpatterns = [path("", index)]
```

Comment:
add the url name here.

Reply:
Done ... finally ))

Reply:
![Screenshot 2023-12-16 at 20 08 44](https://github.com/mate-academy/py-taxi-service-home-page/assets/87115058/9c69054b-9d74-4eb9-ae1c-65698c6ed25a)